### PR TITLE
adding tests for by-id core path

### PR DESCRIPTION
**Pull request by karolinavelkaja**
Original review: https://github.com/Open-CAS/open-cas-linux/pull/539 

This PR needs functionality from: https://github.com/Open-CAS/test-framework/pull/259

Signed-off-by: Karolina Rogowska <karolina.rogowska@intel.com>

### CODE REVIEW

File: test/functional/tests/cache_ops/test_core_add.py [hunk 1]
```diff
@@ -0,0 +1,216 @@
...
+    with TestRun.step(f"Select different links to {core_dev.path} partitions."):
+        selected_links = select_links(core_dev_links)
+
+    with TestRun.step("Start cache and add cores"):
+        paths_from_dmesg = []
+        cache = casadm.start_cache(cache_part, force=True)
+        for i in range(cores_number):
+            core_dev.partitions[i].path = selected_links[i].full_path
+            cache.add_core(core_dev.partitions[i])
+            paths_from_dmesg.append(get_added_core_path_from_dmesg())
```
Comment (arutk) [inline test/functional/tests/cache_ops/test_core_add.py:71]: there is no need to parse dmesg. "casadm -L -b" will print paths exactly as returned by OCF, without any translation.

**Review by arutk — CHANGES_REQUESTED** [review]

File: test/functional/tests/initialize/test_startup_init_config.py [hunk 2]
```diff
@@ -2,24 +2,33 @@
...
-from test_utils.filesystem.file import File
+from test_tools.dd import Dd
 from test_tools.disk_utils import Filesystem
+from test_tools.fs_utils import parse_ls_output, readlink, ls
 from test_utils import fstab
-from test_tools.dd import Dd
+from test_utils.filesystem.file import File
+from test_utils.filesystem.symlink import Symlink
+from test_utils.os_utils import drop_caches, sync
 from test_utils.size import Unit, Size
```
Comment (Deixx) [inline test/functional/tests/initialize/test_startup_init_config.py:26]: could use some cleanup, like removing casadm_parser as a whole module from imports (all the needed functions from casadm_parser are imported) or importing casadm and from casadm

**Review by Deixx — COMMENTED** [review]
Adam's comments still relevant, but this should work almost "as is"

**Comment by katlapinka** [comment]
Moved to https://github.com/Open-CAS/open-cas-linux/pull/1518

**Pull request closed without merging**